openfl-tutorials/interactive_api/Tensorflow_Word_Prediction/director_folder/director_config.yaml

```diff
@@ -0,0 +1,4 @@
+settings:
+  listen_ip: localhost
+  sample_shape: ['3', '96']
+  target_shape: ['10719']
```

Comment:
I test establishing a Federation and Envoys report '10703' target shape for some reason

Reply:
And the third Enovy's target shape is 10702

Reply:
Oh, thank you! It's important and I fixed it in the shard descriptor.